### PR TITLE
Fixes login callbackUrl and redirects to app root not website

### PR DESCRIPTION
**Pull request by agustif**
## What does this PR do?
- Added back callbackUrl only local, change WEBSITE_URL -> APP_URL root.

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?
Try to login adding a custom callback url, shouldn't work.

You should be redirected to app.cal not to call after login

### CODE REVIEW

File: apps/web/pages/auth/login.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ import Button from "@calcom/ui/Button";
 import { EmailField, PasswordField, Form } from "@calcom/ui/form/fields";
 
 import { ErrorCode, getSession } from "@lib/auth";
-import { WEBSITE_URL } from "@lib/config/constants";
+import { WEBSITE_URL, APP_URL } from "@lib/config/constants";
```
Comment (zomars) [inline apps/web/pages/auth/login.tsx:15]: This variable doesn't exist here

File: apps/web/pages/auth/login.tsx [hunk 2]
```diff
@@ -58,13 +58,9 @@ export default function Login({
   };
 
   const telemetry = useTelemetry();
-
-  let callbackUrl = typeof router.query?.callbackUrl === "string" ? router.query.callbackUrl : "/";
-
-  // If not absolute URL, make it absolute
-  if (!/^https?:\/\//.test(callbackUrl)) {
-    callbackUrl = `${WEBSITE_URL}/${callbackUrl}`;
-  }
```
Comment (zomars) [inline apps/web/pages/auth/login.tsx:67]: This will reintroduce a previously fixed vulnerability

**Review by zomars — CHANGES_REQUESTED** [review]

File: apps/web/pages/auth/login.tsx (hunk 2 above)
Comment (agustif) [reply apps/web/pages/auth/login.tsx:67]: I think it's the other way and this reintroduced it.

Previously with only the ternary operator, if router.query.callbackUrl, was a string, it assigned it after the WEBSITE_URL with interpolation, if it was not a string, it just assigned an empty one.

Now if you pass a malicious https:// url, the if regex test never kicks in, and the callbackUrl doesn't get replaced.

I think anyways?

File: apps/web/pages/auth/login.tsx (hunk 1 above)
Comment (agustif) [reply apps/web/pages/auth/login.tsx]: Changed to BASE_URL

**Pull request closed without merging**